types/office-js-preview/index.d.ts

```diff
@@ -618,19 +618,57 @@ declare namespace Office {
     /**
      * Represents an individual tab and the state it should have. For code examples, see  {@link https://docs.microsoft.com/office/dev/add-ins/design/disable-add-in-commands | Enable and Disable Add-in Commands} and {@link https://docs.microsoft.com/office/dev/add-ins/design/contextual-tabs | Create custom contextual tabs}.
      */
-    interface Tab {
+     interface Tab {
         /**
          * Identifier of the tab as specified in the manifest.
          */
         id: string;
         /**
-         * Specifies the controls in the tab, such as menu items, buttons, etc.
+         * Specifies one or more of the controls in the tab, such as menu items, buttons, etc.
+         * 
+         * @remarks
+         * 
+         * When the `Tab` object is part of a {@link Office.RibbonUpdaterData} object passed to the `requestUpdate` method of {@link Office.Ribbon}, this property specifies the IDs of the controls whose enabled status is to be changed. However, if there is a `groups` property on the tab, then this property is ignored and the `controls` properties of the specified groups must be used to change enabled status.
```

Comment:
```suggestion
         * When the `Tab` object is part of a {@link Office.RibbonUpdaterData | RibbonUpdaterData} object passed to the `requestUpdate` method of {@link Office.Ribbon | Ribbon}, this property specifies the IDs of the controls whose enabled status is to be changed. However, if there is a `groups` property on the tab, then this property is ignored and the `controls` properties of the specified groups must be used to change enabled status.
```

Reply:
if you choose not to change to `{@link Office.RibbonUpdaterData | RibbonUpdaterData}`, then I recommend you change the "a" before to "an"